src/runtime/components/InputNumber.vue

```diff
@@ -161,30 +176,32 @@ defineExpose({
       @focus="emitFormFocus"
     />
 
-    <div :class="ui.increment({ class: props.ui?.increment })">
-      <NumberFieldIncrement as-child :disabled="disabled">
+    <div :class="ui.increment({ class: props.ui?.increment })" @click="onIncrement">
```

Comment:
Not sure about this event though, it's not really correct to assume clicking on the div is equal as clicking on the button.